doc/nrf/releases_and_maturity/migration/migration_guide_2.8.rst

```diff
@@ -21,15 +23,81 @@ This document describes the changes required or recommended when migrating your
       * Change1 and description
       * Change2 and description
 
+.. _migration_2.8_required:
+
+Required changes
+****************
+
+The following changes are mandatory to make your application work in the same way as in previous releases.
+
+Build and configuration system
+==============================
+
+.. toggle::
+
+   * Sysbuild now handles the following MCUboot image ID assignments:
+
+     * MCUboot updates (using b0) are automatically assigned to MCUboot.
+	    The ``SB_CONFIG_MCUBOOT_UPDATEABLE_IMAGES`` Kconfig option must not be incremented to include this image.
+     * Applications and MCUboot must use the MCUboot assigned image ID Kconfig values to refer to image IDs instead of hardcoding them.
```

Comment:
*now use. A separate point can be added that applications interacting with the device using MCUboot serial recovery MCUmgr must use the image IDs assigned to them, as well as MCUboot/MCUmgr hooks